webui/react/src/pages/ExperimentDetails/ExperimentCheckpoints.tsx

```diff
@@ -108,15 +107,19 @@ const ExperimentCheckpoints: React.FC<Props> = ({ experiment, pageRef }: Props)
   }, [updateSettings]);
 
   const stateFilterDropdown = useCallback(
-    (filterProps: FilterDropdownProps) => (
-      <TableFilterDropdown
-        {...filterProps}
-        multiple
-        values={settings.state}
-        onFilter={handleStateFilterApply}
-        onReset={handleStateFilterReset}
-      />
-    ),
+    (filterProps: FilterDropdownProps) => {
+      if (!settings.state) return <Spinner spinning />;
```

Comment:
Does that mean we keep having spinner if state is not defined?
I think state is fixed value, so idk if we need this